R/datamapping-groupings.R

```diff
@@ -14,16 +14,17 @@ Groupings <- R6::R6Class(
                               fill = NULL,
                               linetype = NULL,
                               shape = NULL) {
-      if (!"GroupMapping" %in% class(color)) {
+
+      if (isOfType(color, "GroupMapping")) {
         color <- GroupMapping$new(group = color)
       }
-      if (!"GroupMapping" %in% class(fill)) {
+      if (isOfType(fill, "GroupMapping")) {

```

Comment:
FYI. You can also use isOfType(fill, GroupMapping) (Without "" using type directly)
Also the code is wrong. Is should be `!isOfType(xxx)`